src/virtualenv/config/env_var.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import, unicode_literals
+
+import os
+
+from .convert import convert
+
+
+def get_env_var(key, as_type):
+    """Get the environment variable option.
+
+    :param key: the config key requested
+    :param as_type: the type we would like to convert it to
+    :return:
+    """
+    environ_key = "VIRTUALENV_{}".format(key.upper())
+    if environ_key in os.environ:
```

Comment:
Probably? it doesn't matter, but there are silent unicode conversions happening here (on Py2), becuase you've used unicode_literals everywhere (personally I recommend never using that, but I assume you have an opinion there). I don't think anything blows up here unless for some silly reason key is non-ASCII I guess, so possibly it doesn't matter, but pointing it out just in case.

Reply:
I'll cast the key to string 😄